legal-api/src/legal_api/models/configuration.py

```diff
@@ -64,6 +82,12 @@ def validate_value(self):
                 int(self.val)
             except ValueError as exc:
                 raise ValueError(f'Value for key {self.name} must be an integer') from exc
+            num_dissolutions_allowed = self.val if self.name == 'NUM_DISSOLUTIONS_ALLOWED'\
+                else Configuration.find_by_name('NUM_DISSOLUTIONS_ALLOWED').val
+            max_dissolutions_allowed = self.val if self.name == 'MAX_DISSOLUTIONS_ALLOWED'\
+                else Configuration.find_by_name('MAX_DISSOLUTIONS_ALLOWED').val
+            if int(num_dissolutions_allowed) > int(max_dissolutions_allowed):
+                raise ValueError('NUM_DISSOLUTIONS_ALLOWED cannot be greater than MAX_DISSOLUTIONS_ALLOWED.')
```

Comment:
I don't think you should be doing this type of validation here.

We should do it further up where the resource is called.

Reply:
Updated